src/presentation/components/header/ThemeButton.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <Button
+    class="header__theme"
+    :text="t('home.theme')"
+    :icon="IconReplace"
+    @click="changeTheme"
+  />
+</template>
+
+<script setup lang="ts">
+import { useColorMode } from '@vueuse/core';
+import { IconReplace } from '@codexteam/icons';
+import { useI18n } from 'vue-i18n';
+import Button from '@/presentation/components/button/Button.vue';
+
+const { t } = useI18n();
+const mode = useColorMode();
+
+/**
+ * Change theme from dark/light to light/dark
+ */
+function changeTheme() {
+  mode.value = useColorMode().value === 'light' ? 'dark' : 'light';
```

Comment:
will it be stored somewhere and accepted on the second visit?

Reply:
Yep, it's in the local storage